cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -342,3 +426,14 @@ func isAutoProvisioning(spc *apis.StoragePoolClaim) bool {
 func isManualProvisioning(spc *apis.StoragePoolClaim) bool {
 	return spc.Spec.BlockDevices.BlockDeviceList != nil
 }
+
+// isSPCDeletionCandidate return true when deletion timestamp & finalizer is
+// available on spc
+func isSPCDeletetionCandidate(spc *apis.StoragePoolClaim) bool {
```

Comment:
If at all we need this -- better to put it in a method as this behaviour/decision is only for SPC. Yes, I know the old code has some inherent code like this.